src/main/java/com/netuitive/ananke/statsd/client/StatsDClient.java

```diff
@@ -0,0 +1,36 @@
+package com.netuitive.ananke.statsd.client;
+
+import com.netuitive.ananke.statsd.client.request.DecrementRequest;
+import com.netuitive.ananke.statsd.client.request.EventRequest;
+import com.netuitive.ananke.statsd.client.request.GaugeRequest;
+import com.netuitive.ananke.statsd.client.request.HistogramRequest;
+import com.netuitive.ananke.statsd.client.request.IncrementRequest;
+import com.netuitive.ananke.statsd.client.request.ServiceCheckRequest;
+import com.netuitive.ananke.statsd.client.request.SetRequest;
+import com.netuitive.ananke.statsd.client.request.TimedRequest;
+import com.netuitive.ananke.statsd.client.request.TimingRequest;
+import java.io.IOException;
+
+/**
+ *
+ * @author john.king
+ */
+public interface StatsDClient {
+    public void decrement(DecrementRequest req) throws IOException;
```

Comment:
Not sure we want to have those IOException defined in the interface. It put burden on the user code to catch those exception and don't know how to deal with it.